### ISSUE TITLE
Commits to Github not updating in TX project

### ISSUE THREAD

**Issue by VilliGunn** [transifex/txgh#63]
When translation files are updated and pushed to Github the changes to the files are not reflected  in the TX project.

Dockerfile:
```
FROM mjjacko/txgh:latest

ENV RACK_ENV production
ENV GITHUB_PUSH_SOURCE_TO "icelandair-meals"
ENV GITHUB_USERNAME "VilliGunn"
ENV GITHUB_TOKEN "******"
ENV GITHUB_WEBHOOK_SECRET "******"
ENV GITHUB_BRANCH develop
ENV TX_CONFIG_PATH config/tx.config
ENV TX_USERNAME "tmsoftware"
ENV TX_PASSWORD "******"
ENV TX_PUSH_TRANSLATIONS_TO "Icelandair/micro.translations"
ENV TX_WEBHOOK_SECRET "******"

COPY /config/tx.config /usr/app/config/tx.config

EXPOSE 9292
CMD ["puma", "-p", "9292"]
```

tx.config:
```
[main]
host = https://www.transifex.com
lang_map =

# Create one such section per resource
[icelandair-meals.mealsformenglishjson]
file_filter = translations/<lang>/sample.json
source_file = sample.json
source_lang = en
type = JSON
```

the txgh logs:
```
INFO -- : Processing request at /hooks/github
INFO -- : processing payload from request.body
INFO -- : request github branch: heads/develop
INFO -- : config github branch: heads/develop
INFO -- : found branch in github request
INFO -- : processing resource
INFO -- : processing commit
INFO -- : processing modified file: translations/en_GB/sample.json
```
 
Everything works the other way around, from TX to github when everything is translated and/or reviewed.

**Comment by matthewjackowski** [comment]
@VilliGunn So currently Txgh doesn't support translation 'syncing'.  In other words the translation file flow is one way from Tx to Github and only source files can be pushed from GitHub to Tx.
 
However it sounds like you might have an interesting workflow that I have not considered.  
If we added support to push translation files back to Tx, what would expect to happen to the existing files in Tx?
In other words, would you expect the translations pushed to GitHub to overwrite Tx?  Or would you need some type of review step?

**Comment by VilliGunn** [comment]
@matthewjackowski Actually we just wanted to sync the source language file from GitHub to TX and the problem was we only had language files and no source file. 

Txgh never pushed the source language file to our repo which caused our confusion. 

We manually added the source language file to our repo and pushed, changed and pushed again and then the changes were reflected in our TX project.

This is what we originally wanted. We do not txgh to sync the translation files from GitHub to TX.
